apiserver/README.md

```diff
@@ -38,15 +38,15 @@ curl --silent -X 'GET' \
     -H 'accept: application/json'
 ```
 
-### Helm
+### Install with Helm
 
 Make sure the version of Helm is v3+. Currently, [existing CI tests](https://github.com/ray-project/kuberay/blob/master/.github/workflows/helm-lint.yaml) are based on Helm v3.4.1 and v3.9.4.
 
 ```sh
 helm version
 ```
 
-### Install KubeRay Operator
+#### Install KubeRay Operator
```

Comment:
curious, why do we make kuberay operator installation a subsection for helm installation?